.github/workflows/release.yml

```diff
@@ -52,31 +49,22 @@ jobs:
       - name: Compile Windows
         if: matrix.name == 'windows-amd64'
         # On Windows we only build using the GNU toolchain (not MSVC, the default)
-        run: cargo build --release --target ${{ matrix.toolchain }}
-      - name: Compile staticlib
-        if: contains(matrix.toolchain, 'musl')
-        # Linux requires a different toolchain for static lib generation
+        shell: bash
         run: |
           cargo build --release --target ${{ matrix.toolchain }}
-          # overwrite static glibc static lib
-          mv target/${{ matrix.toolchain }}/release/libed25519_bip32.a target/release
+          mv target/${{ matrix.toolchain }}/release/*.dll target/release
+          mv target/${{ matrix.toolchain }}/release/*.a target/release
       - name: Prepare files
-        if: matrix.name != 'windows-amd64'
+        shell: bash
         run: |
           mkdir artifacts
-          mv LICENSE ed25519_bip32.h target/release/libed25519_bip32.${{ matrix.extension }} target/release/libed25519_bip32.a artifacts
+          # copy all relevant headers and static and dynamic libs into the new directory
+          find . -maxdepth 3 -type f \( -path './LICENSE' -o -path '*.h' -o -path './target/release/*.a' -o -path './target/release/*.dll' -o -path './target/release/*.dylib' -o -path './target/release/*.so' \) -print0 | xargs -0 -I {} mv {} artifacts
           cd artifacts
-          tar -czf libed25519_bip32_${{ matrix.name }}.tar.gz *
-          mv libed25519_bip32_${{ matrix.name }}.tar.gz ..
-      - name: Prepare files Windows
-        if: matrix.name == 'windows-amd64'
-        run: |
-          mkdir artifacts
-          Move-Item -Path LICENSE, ed25519_bip32.h -Destination artifacts
-          Move-Item -Path target/${{ matrix.toolchain }}/release/ed25519_bip32.dll -Destination artifacts
-          Move-Item -Path target/${{ matrix.toolchain }}/release/libed25519_bip32.a -Destination artifacts
-          Compress-Archive -Path artifacts/* -Destination ${{ matrix.artifact }}
+          tar -czf ${{ env.ARTIFACT_NAME }}_${{ matrix.name }}.tar.gz *
+          mv ${{ env.ARTIFACT_NAME }}_${{ matrix.name }}.tar.gz ..
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
-          files: ${{ matrix.artifact }}
\ No newline at end of file
+          files: ${{ env.ARTIFACT_NAME }}_${{ matrix.name }}.tar.gz
+          tag_name: v0.0.1
```

Comment:
I'm not sure how `tag_name` works. Will this create a release for the same tag on every run or update the tag to point to the commit that triggered the workflow?

Reply:
same, this is temporary